drivers/shared/executor/executor_linux.go

```diff
@@ -532,28 +533,25 @@ func (l *LibcontainerExecutor) handleExecWait(ch chan *waitResult, process *libc
 	ch <- &waitResult{ps, err}
 }
 
-func configureCapabilities(cfg *lconfigs.Config, command *ExecCommand) error {
-	// TODO(shoenig): allow better control of these
-	// use capabilities list as prior to adopting libcontainer in 0.9
-
-	// match capabilities used in Nomad 0.8
-	if command.User == "root" {
-		allCaps := SupportedCaps(true)
+func configureCapabilities(cfg *lconfigs.Config, command *ExecCommand) {
+	switch command.User {
+	case "root":
+		// when running as root, use the legacy set of system capabilities, so
+		// that we do not break existing nomad clusters using this "feature"
```

Comment:
This is unfortunate. The good thing is that allowing `root` in exec tasks is an opt-in behavior. We can also change it and provide the legacy configuration flag.